test/test_server.js

```diff
@@ -85,10 +85,10 @@ TestServer.prototype.stop = function () {
   if (this.server) {
     const doomed = [
       this.server.close(),
-      this.mail.close()
+      this.mail.close
```

Comment:
@deeptibaghel we probably need these function calls here, any reason to remove them?